microsoft-edge/devtools-guide-chromium/whats-new/2022/01/devtools.md

```diff
@@ -0,0 +1,93 @@
+## Introducing the Detached Elements tool
+
+<!-- Title: Check out the new Detached Elements tool -->
+<!-- Subtitle: The new Detached Elements tool is now available by default in Microsoft Edge 97. -->
+
+A DOM node is considered _detached_ when it's no longer attached to any element of the DOM, but is still being retained in memory by Microsoft Edge.  The browser cannot garbage-collect the detached element, because some JavaScript object is still referencing the element, even though the element is no longer on the page or is no longer a part of the DOM.
+
+The new **Detached Elements** tool finds all of the detached elements on your page and displays them.  You can expand and collapse a detached element to see the parent and child nodes that are also being retained.  You can trigger the browser's garbage collection by clicking the **Collect garbage** icon, and then confirm that you have a memory leak when a detached element cannot be garbage-collected.  To jump into the JavaScript code that's referencing the detached element, click the **Analyze** button to take a heap snapshot.
+
+The **Detached Elements** tool was initially available as an experiment in [Microsoft Edge version 93](../../2021/07/devtools.md#debug-dom-node-memory-leaks-with-the-new-detached-elements-tool).  The tool is now available by default in Microsoft Edge version 97.
+
+![The Detached Elements tool.](../../media/2022/01/detached-elements-tool.png)
+
+See also:
+* [Debug DOM memory leaks with the Detached Elements tool](../../../memory-problems/dom-leaks.md)
+* [Debug memory leaks with the Microsoft Edge Detached Elements tool - YouTube](https://www.youtube.com/watch?v=v2iy17ptmBk&ab_channel=MicrosoftEdge)
+
+
+<!-- ====================================================================== -->
+## Microsoft Edge DevTools extension for Visual Studio Code
+
+For general information about this extension, see [Microsoft Edge DevTools extension for Visual Studio Code](../../../../visual-studio-code/microsoft-edge-devtools-extension.md).
+
+### Screencast enhancements, deprecation warnings, and new launch options for Microsoft Edge
+
+<!-- Title: Screeencast improvements and launch options for Microsoft Edge in the Visual Studio Code extension -->
+<!-- Subtitle: The correct list of emulated devices is shown for the screeencast, the correct device emulation is displayed, and there are now launch arguments for the browser. -->
+
+The current version of the Microsoft Edge DevTools extension for Visual Studio Code fixes several issues that were reported by the community:
+
+*  The list of emulated devices in the screencast is now the same as the list of emulated devices in the browser.
+
+*  Device emulation now adds the correct user agent string, to trigger the correct display.
+
+*  You now get a warning when you use a deprecated launch version of Microsoft Edge, indicating that you'd miss out on features by using a deprecated version of Microsoft Edge.
```

Comment:
Let's remove the "by using a deprecated version of Microsoft Edge." as this is about how Edge is launched from the VS Code side, this isn't about the version of Edge. And it means we don't repeat "deprecated".